src/pages/blog/2020-04-19-findings-of-emnlp.md

```diff
@@ -5,17 +5,19 @@ date: 2020-04-19
 updated: 
   - date: 2020-04-26
     details: "Created addendum with FAQs"
+  - date: 2020-05-25
+    details: "New terminology: 'companion' instead of 'journal'"
```

Comment:
```suggestion
  - date: 2020-05-25
    details: "New terminology: 'companion' instead of 'journal'"
  - date: 2020-04-26
    details: "Created addendum with FAQs"
```